src/main/resources/messages_ua.properties

```
@@ -47,6 +47,10 @@ empty.password=\u041E\u0442\u0440\u0438\u043C\u0430\u043D\u043E \u043F\u0443\u04
 empty.confirm.password=\u041F\u0430\u0440\u043E\u043B\u044C \u043F\u0456\u0434\u0442\u0432\u0435\u0440\u0434\u0436\u0435\u043D\u043D\u044F \u043F\u0443\u0441\u0442\u0438\u0439
 empty.email=\u041E\u0442\u0440\u0438\u043C\u0430\u043D\u0443 \u043F\u0443\u0441\u0442\u0435 \u0437\u043D\u0430\u0447\u0435\u043D\u043D\u044F \u0456\u043C\u0435\u0439\u043B\u0443
 
+blank.topic=\u041D\u0430\u0437\u0432\u0430 \u0441\u043A\u043B\u0430\u0434\u0430\u0454\u0442\u044C\u0441\u044F \u043B\u0438\u0448\u0435 \u0437 \u043F\u0440\u043E\u0431\u0456\u043B\u0456\u0432
+blank.description=\u041E\u043F\u0438\u0441 \u0441\u043A\u043B\u0430\u0434\u0430\u0454\u0442\u044C\u0441\u044F \u043B\u0438\u0448\u0435 \u0437 \u043F\u0440\u043E\u0431\u0456\u043B\u0456\u0432
+blank.wishes.to.exchange=\u0412\u043F\u043E\u0434\u043E\u0431\u0430\u043D\u043D\u044F \u0434\u043E \u043E\u0431\u043C\u0456\u043D\u0443 \u0441\u043A\u043B\u0430\u0434\u0430\u0454\u0442\u044C\u0441\u044F \u043B\u0438\u0448\u0435 \u0437 \u043F\u0440\u043E\u0431\u0456\u043B\u0456\u0432
```

The detailed response messages are also applicable to the UA locale:
```suggestion
blank.topic=Заголовок не має бути пустим
blank.description=Опис не має бути пустим
blank.wishes.to.exchange=Бажані пропозиції на обмін не мають бути пустими
```